imports/ui/components/news/news_item.html

```diff
@@ -31,4 +33,6 @@ <h3 class='comments__header'>Comments</h3>
   {{/if}}
     </div>
     {{/with}}
+    {{/if}}
+    {{/if}}
```

Comment:
This code smell of this double-if syntax has always bothered me. I think a good thing to put on the someday/maybe list would be refactoring these into convenience functions if there's no better idiom in Handlebars... Or just rewrite the whole thing in React, I guess.